elixir/bank-account/lib/account.exs

```diff
@@ -0,0 +1,50 @@
+defmodule Account do
+  use GenServer
+
+  #Client
+  def open_bank do
+    with {:ok, pid} <- GenServer.start_link(__MODULE__, %{balance: 0, status: :open}) do
+      pid
+    else
+      error -> raise "Account was not opened. #{inspect(error)}"
+    end
+  end
+
+  def update(pid, amount) do
+    GenServer.cast(pid, {:deposit, amount})
+  end
+
+  def view(pid) do
```

Comment:
You don't need to use it of course, but did you know that there is an is_pid guard?
https://hexdocs.pm/elixir/1.12/Kernel.html#is_pid/1

Reply:
I didn't guard on the pid but on the amount in the bank_account file